addons/binding/pom.xml

```diff
@@ -77,6 +77,7 @@
     <module>org.openhab.binding.meteostick</module>
     <module>org.openhab.binding.miele</module>
     <module>org.openhab.binding.mihome</module>
+    <module>org.openhab.binding.miio</module>
```

Comment:
Please also add the binding to the Karaf feature.xml file.

Reply:
Yes, added the line for the feature